android-components/components/support/webextensions/src/main/java/mozilla/components/support/webextensions/WebExtensionSupport.kt

```diff
@@ -174,6 +174,7 @@ object WebExtensionSupport {
         onSelectTabOverride: ((WebExtension?, String) -> Unit)? = null,
         onUpdatePermissionRequest: onUpdatePermissionRequest? = { _, _, _, _ -> },
         onExtensionsLoaded: ((List<WebExtension>) -> Unit)? = null,
+        onDisabledExtensionProcessSpawning: (() -> Unit)? = null,
     ) {
         this.onUpdatePermissionRequest = onUpdatePermissionRequest
         this.onExtensionsLoaded = onExtensionsLoaded
```

Comment:
**No a blocker but It would be nice to investigate**: Noticed for all other callbacks we are also creating an [internal reference](https://github.com/mozilla-mobile/firefox-android/blob/9e5d0037b4e0afe5f02e5d56b0ac36e66c41ca62/android-components/components/support/webextensions/src/main/java/mozilla/components/support/webextensions/WebExtensionSupport.kt#L63-L65), don't remember exactly why,  I think it will be good to double if we don't need to follow the same pattern here.

Reply:
Good catch! Now that I have (PR 3472)[https://github.com/mozilla-mobile/firefox-android/pull/3472] I can actually remove this since we'll just send a `BrowserAction` to the store. This will work the same as `onExtensionListUpdated()` in dispatching to the store from within this class.